test/v8-to-istanbul.js

```diff
@@ -28,6 +30,28 @@ describe('V8ToIstanbul', () => {
+      const source =`var cov_1myqaytex8=function(){var path="./test/fixtures/scripts/branches.js";var hash="05fd1c5758177cf8534c86e9665d000812274ffa";var global=new Function("return this")();var gcv="__coverage__";var coverageData={path:"./test/fixtures/scripts/branches.js",statementMap:{"0":{start:{line:2,column:10},end:{line:2,column:18}},"1":{start:{line:5,column:10},end:{line:5,column:37}},"2":{start:{line:8,column:10},end:{line:8,column:31}},"3":{start:{line:11,column:0},end:{line:16,column:3}},"4":{start:{line:12,column:12},end:{line:12,column:14}},"5":{start:{line:15,column:4},end:{line:15,column:25}},"6":{start:{line:20,column:2},end:{line:24,column:3}},"7":{start:{line:21,column:4},end:{line:21,column:23}},"8":{start:{line:23,column:14},end:{line:23,column:16}},"9":{start:{line:27,column:0},end:{line:27,column:3}},"10":{start:{line:30,column:10},end:{line:31,column:10}}},fnMap:{"0":{name:"e",decl:{start:{line:19,column:9},end:{line:19,column:10}},loc:{start:{line:19,column:14},end:{line:25,column:1}},line:19}},branchMap:{"0":{loc:{start:{line:2,column:10},end:{line:2,column:18}},type:"binary-expr",locations:[{start:{line:2,column:10},end:{line:2,column:12}},{start:{line:2,column:16},end:{line:2,column:18}}],line:2},"1":{loc:{start:{line:5,column:10},end:{line:5,column:37}},type:"cond-expr",locations:[{start:{line:5,column:18},end:{line:5,column:25}},{start:{line:5,column:28},end:{line:5,column:37}}],line:5},"2":{loc:{start:{line:8,column:10},end:{line:8,column:31}},type:"binary-expr",locations:[{start:{line:8,column:10},end:{line:8,column:11}},{start:{line:8,column:15},end:{line:8,column:16}},{start:{line:8,column:20},end:{line:8,column:25}},{start:{line:8,column:29},end:{line:8,column:31}}],line:8},"3":{loc:{start:{line:11,column:0},end:{line:16,column:3}},type:"if",locations:[{start:{line:11,column:0},end:{line:16,column:3}},{start:{line:11,column:0},end:{line:16,column:3}}],line:11},"4":{loc:{start:{line:20,column:2},end:{line:24,column:3}},type:"if",locations:[{start:{line:20,column:2},end:{line:24,column:3}},{start:{line:20,column:2},end:{line:24,column:3}}],line:20},"5":{loc:{start:{line:30,column:10},end:{line:31,column:10}},type:"binary-expr",locations:[{start:{line:30,column:10},end:{line:30,column:12}},{start:{line:31,column:2},end:{line:31,column:4}},{start:{line:31,column:8},end:{line:31,column:10}}],line:30}},s:{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0},f:{"0":0},b:{"0":[0,0],"1":[0,0],"2":[0,0,0,0],"3":[0,0],"4":[0,0],"5":[0,0,0]},_coverageSchema:"43e27e138ebf9cfc5966b082cf9a028302ed4184",hash:"05fd1c5758177cf8534c86e9665d000812274ffa"};var coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}return coverage[path]=coverageData;}();// basic binary operation.
+const a=(cov_1myqaytex8.s[0]++,(cov_1myqaytex8.b[0][0]++,99)||(cov_1myqaytex8.b[0][1]++,33));// basic conditional operation.
```

Comment:
I haven't done any digging yet, but I bet you could make a more contrived example where you just have a file with:

```js
const foo.js
// sourceMap=data=abc123
```

☝️ I'd then just put this in the fixtures directory, and have it so it remaps to another file in the fixtures directory.

Reply:
fair point about a smaller file
however, it still has to be generated since the sourceRoot is an absolute path